subjects/printmiddle/README.md

```diff
@@ -0,0 +1,22 @@
+## print-middle
+
+### Instructions
+
+Write a program that prints the middle argument of the command line. 
+- If the number of arguments is less than 1, print (`'\n'`)
+- If the number of arguments is **even**, print the middle two arguments with a space (`' '`) between the arguments
+- If the number of arguments is **odd**, print the middle one.
+- Print (`'\n'`) at the end of the output.
+
+### Usage
+
+```console
+$ go run . | cat -e
+$
```

Comment:
I think it is better to remove the '$' after the output, as the function does not return that character.

Reply:
yes, you right, but `cat  -e` show if there's any new line after the output.
and the subject says that should add a new line after the output .

Reply:
oh yes, sorry for my mistake, you are right, I was running without the cat -e. Let's kept as is.